### ISSUE TITLE
Warning message on Layout Add form not always displayed

### ISSUE THREAD

**Issue by ghost** [backdrop/backdrop-issues#4845]
**Description of the bug**

A warning is displayed on `/admin/structure/layouts/add` if you enter a path like `node/2`, informing you to use `node/%` with visibility conditions instead. However this warning doesn't always appear, and seems to be dependent on other fields. The warning is:

> The path entered will create a new page and disable the current "node/2". If you intended to add a layout for "node/2", use the path "node/%" and add a URL path condition for "node/2".

**Steps To Reproduce**

To reproduce the behavior:
1. Go to `/admin/structure/layouts/add`
2. Add a Layout Name
3. *Do not* click on a Layout Template
4. Enter a Path of `node/2`

**Actual behavior**

No warning displayed, just an error about the Layout Template being required (which is weird in itself since we haven't tried to submit the form yet).

**Expected behavior**

The warning should be displayed, regardless of what other fields have been entered already.

**Additional information**

Possibly related to https://github.com/backdrop/backdrop-issues/issues/4555

**Comment by klonos** [comment]
> ...just an error about the Layout Template being required (which is weird in itself since we haven't tried to submit the form yet).

This has always been the case. I do find it odd UX, but never noticed that this also prevents the other error re existing path from being shown (which sounds like a bug).

**Comment by BWPanda** [comment]
Yeah, I'm wondering if we should re-purpose this issue to prevent validation happening before submitting the form...? If we do that, it'll fix this issue along with another problem (not sure if there's an issue for it) where you can't add contexts or visibility conditions before selecting a template...
